package.xml

```diff
@@ -0,0 +1,3908 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
what is this file for ?
sounds like it should be removed and added it to gitigonre